pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "pikaraoke"
+version = "1.0.0"
+description = "Youtube-based Karaoke machine for Raspberry Pi, OSX, Windows, and Linux"
+authors = ["Marius <my_email@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+CherryPy = "18.10.0"
+Flask = "2.2.5"
+qrcode = "7.4.2"
+psutil = "6.0.0"
+unidecode = "1.3.8"
+requests = "2.32.3"
+flask-paginate = "2021.10.29"
+jinja2 = "3.0.3"  # Avoid escape issue: https://github.com/sphinx-doc/sphinx/issues/10289
+Babel = "2.9.1"
+Flask-Babel = "2.0.0"
+selenium = "4.22.0"
```

Comment:
This version of selenium is not compatible with pikaraoke, per: https://github.com/vicwomg/pikaraoke/issues/345

Please use 4.21.0

Reply:
Will downgrade

Reply:
Downgraded. I rebase to keep the commits clean. Hope it's not too annoying to keep track of the review. Let me know.